app/src/main/java/com/mercadolibre/android/mlbusinesscomponentsapp/MainActivity.java

```diff
@@ -33,6 +34,12 @@ protected void onCreate(Bundle savedInstanceState) {
         MLBusinessLoyaltyHeaderView loyaltyHeaderView = findViewById(R.id.loyaltyHeaderView);
         LinearLayout benefitContainer = findViewById(R.id.loyaltyBenefitsContainer);
 
+        Button button = findViewById(R.id.buttonOpen);
+        button.setOnClickListener(v -> {
```

Comment:
Coti, esto se puede cambiar por
` button.setOnClickListener {
      Intent intent = new Intent(MainActivity.this,BotonesActivity.class);
      startActivity(intent);
}`